### PR TITLE
Limit number of crop rendering containers

### PR DESCRIPTION
**Pull request by Elfoniok**

### CODE REVIEW

File: apps/blender/task/verifier.py [hunk 1]
```diff
@@ -161,9 +162,15 @@ def _crop_rendered(self, results, time_spend, verification_context):
             logger.warning("Wrong main program file: %r", err)
             src_code = ""
 
+        self.docker_manager = DockerManager.install()
```
Comment (badb) [inline apps/blender/task/verifier.py:165]: I don't think that's the right approach. Can we at least use DockerTaskThread here?

File: apps/blender/task/blendercropper.py [hunk 2]
```diff
@@ -134,16 +136,43 @@ def render_crops(self, computer, resources, crop_rendered,
...
+                              computer, resources,
+                              subtask_info, crop_rendered,
+                              crop_render_failure)
         return self.crop_size
 
+    def render_next_crop(self, results, time_spend, verification_context,
+                         crops_info, crops_path, computer, resources,
+                         subtask_info, crop_rendered,
+                         crop_render_failure):
+        if results and time_spend and verification_context:
```
Comment (badb) [inline apps/blender/task/blendercropper.py:150]: It's not clear what is happening here and in which situation these 3 arguments should be None. Can you refactor or at least add some comments?

**Review by badb — CHANGES_REQUESTED** [review]

File: apps/blender/task/verifier.py (hunk 1 above)
Comment (Elfoniok) [reply apps/blender/task/verifier.py:165]: Docker Task Thread is well for Task, its not well prepared to handle something else, but i try to make it nicer.

**Comment by Elfoniok** [comment]
Closing in favor of https://github.com/golemfactory/golem/pull/2306

**Pull request closed without merging**